tests/conftest.py

```diff
@@ -234,7 +232,7 @@ def build_header(path, method, role='submitter'):
 
 @pytest.fixture(scope='session')
 def dictionary_setup():
```

Comment:
I think this fixture should actually take in the app fixture. That way you don't have to modify all the test calls. And it will force you to make the app fixture session scoped.

Reply:
It does not. I tried ):

Reply:
Right, was the error was mixing scope of fixtures? How can something that runs once per test session take something that runs once every test?